src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
```

Comment:
We should probably call `.trim` on `entryData` *before* we check it against the list of valid identifiers, rather than in [line 56](https://github.com/netlify/netlify-cms/pull/460/files#diff-bb12caf37386494eb84dc9a5aca4abe1R56), so we don't reject field names based on spurious whitespace.